src/rendition-mixin.js

```diff
@@ -87,7 +92,8 @@ let renditionSelectionMixin = function(hlsHandler) {
     return playlists
       .master
       .playlists
-      .map((e, i) => new Representation(hlsHandler, e, i));
+      .filter((media) => !isBlacklisted(media))
```

Comment:
I think the current decision is that the current solution is OK for now, and we can revisit it later.